shotover-proxy/tests/kafka_int_tests/test_cases.rs

```diff
@@ -549,6 +551,159 @@ pub async fn produce_consume_partitions1_kafka_node_goes_down(
     }
 }
 
+pub async fn produce_consume_partitions1_shotover_nodes_go_down(
+    driver: KafkaDriver,
+    shotover_nodes_to_kill: &mut VecDeque<BinProcess>,
+    connection_builder: &KafkaConnectionBuilder,
+    topic_name: &str,
+) {
+    if driver.is_cpp() {
```

Comment:
Lets remove this since we've disabled the cpp driver from running this at the test level?